CMakeLists.txt

```diff
@@ -160,7 +163,7 @@ endif()
 
 add_library(metal_header_directories INTERFACE)
 target_include_directories(metal_header_directories INTERFACE ${PROJECT_SOURCE_DIR}/tt_metal/hw/inc)
-target_include_directories(metal_header_directories SYSTEM INTERFACE ${reflect_SOURCE_DIR})
+target_include_directories(metal_header_directories SYSTEM INTERFACE ${reflect_SOURCE_DIR} ${flatbuffers_SOURCE_DIR}/include ${nanomsg_SOURCE_DIR}/include)
```

Comment:
Do these libs not have an interface for include directories?

Reply:
NNG does not, but flatbuffers provides one: `FlatBuffers::FlatBuffers`. 

Linking it would require an extra `target_link_libraries` line and it might be cleaner the way it is right now.